src/viewer.js

```diff
@@ -1680,6 +1699,18 @@ $.extend( $.Viewer.prototype, $.EventSource.prototype, $.ControlDock.prototype,
         return this;
     },
 
+    /**
+     * Force the viewer to reset it's size to it's container.
+     * @param Boolean ignoreAutorResizeSetting (default: false) If true, forces the resize even if Viewer.autoResize==false
+     * @returns {OpenSeadragon.Viewer} Chainable.
+     */
+     forceResize: function(ignoreAutoResizeSetting) {
```

Comment:
I feel like, given the name `forceResize`, it should always resize, regardless of the auto resize setting.

Reply:
I updated the name to `forceImmediateResize`, to emphasize why it is (hypothetically) useful. I am not sure whether it needs the parameter or not; that relates to another question you asked about the intended usage of this (if would it need to be called manually). I've removed it, for now at least.